stdlib/tkinter/ttk.pyi

```diff
@@ -1023,29 +1021,35 @@ class Treeview(Widget, tkinter.XView, tkinter.YView):
         anchor: tkinter._Anchor = ...,
         # id is read-only
     ) -> _TreeviewColumnDict | None: ...
-    def delete(self, *items: str) -> None: ...
-    def detach(self, *items: str) -> None: ...
-    def exists(self, item: str) -> bool: ...
+    @overload
+    def delete(self, __item: str | int) -> None: ...
+    @overload
+    def delete(self, *items: str | int) -> None: ...
+    @overload
+    def detach(self, __item: str | int) -> None: ...
+    @overload
+    def detach(self, *items: str | int) -> None: ...
```

Comment:
@brccabral and @Akuli, do we need the overloads here? Are there any valid calls that would be disallowed if we just did:

```suggestion
    def delete(self, *items: str | int) -> None: ...
    def detach(self, *items: str | int) -> None: ...
```

Reply:
probably not, I had created the overload to accept lists, but they are not valid... I'll edit a new commit